static/js/countdown.js

```diff
@@ -51,9 +46,7 @@
 
 	if (typeof(GBCountdown) === "undefined") {
 		window.GBCountdown = define_gbcountdown();
-	} else {
-		console.log("GBCountdown already defined.");
```

Comment:
I would leave this one in, this one actually has some purpose  (as oposed to the other ones I forgot to remove :blush: )